tests/cli/wit-directory-output-valid.wit

```diff
@@ -1,14 +0,0 @@
-// RUN: component wit % --out-dir %tmpdir | component wit %tmpdir
-
-package a:b {
```

Comment:
Mind transforming this test to be valid with a `package foo:bar;` at the top?

Reply:
This test has the same uncertainties as the one below testing the `--out-dir` flag.